committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+# SIG Governance Template
+
+## Goals
+
+The following document outlines a template governance structure for
+SIGs to follow.  Its goals are to provide the necessary structure
+for SIGs to self organize and self govern.
+
+This document is focussed on:
+
+- Outlining organizational responsibilities
+- Outlining organizational roles
+- Outlining processes and tools
+
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of project leadership
+- The process for making technical decisions
+- The process for ensuring continued test health
+
+## How to use this template
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+**Note:** Specific responsibilities towards the project are required
+some SIGs - e.g. ensuring test health.
+
+FAQ:
+
+- [Why is it so long?](#why-is-it-so-long)
+- [Why so much structure?](#why-so-much-structure)
+- [Why use mailing lists for tracking roles?](#why-use-mailing-lists-for-tracking-roles)
+- [Why track membership as a special role?](#why-track-membership-as-a-special-role)
+- [How do the results from the survey fit in?](#how-do-the-results-from-the-survey-fit-in)
+
+---
+
+# SIG ~~Foo~~ Governance
+
+The following document outlines the governance for
+SIG ~~Foo~~.  Its goals are to define the necessary structure for
+SIG ~~Foo~~ to self govern and to define leadership responsibilities
+within the SIG.
+
+This document is broken up into the following parts:
+
+- Roles within the SIG and their responsibilities
+- Processes used by members of the SIG
+
+## Roles and responsibilities
+
+This section describes the various roles governing the SIG.
+
+Titles are not honorific, but critical to the functioning of the SIG.
+
+### Project leads
+
+Canonical list of project leads: kubernetes-sig-~~foo~~-project-leads@googlegroups.com
+This list may be used to escalate issues to project leads.
+
+Project leads:
+
+- Run [operational concerns][business-operations] in a SIG.
+- May also be Technical Leads, however the membership
+and responsibilities of each role is separate.
+- May retain the title "~~Foo~~ project lead emeritus" when stepping
+  down.
+
+While project leads are ultimately responsible for these items, there
+is an expectation that all members of the SIG will contribute
+in these areas.  Project leads may ask for volunteers from the other
+members assist with specific responsibilities.
+
+#### Responsibilities towards members the SIG
+
+- Acting as role model within the SIG
+  - Other members will look up to project leads.  Leads
+    must model respectful behavior towards all members of the community
```

Comment:
nit: decide on a line wrapping it varies for list items.

Reply:
Good eye